caramel/scripts/tool.py

```diff
@@ -115,11 +115,17 @@ def csr_clean(number):
         CSR = models.CSR.query().get(number)
         if not CSR:
             error_out("ID not found")
-        certs = sorted(CSR.certificates, key=lambda cert: cert.id)
+        certs = sorted(CSR.certificates, key=lambda cert: cert.not_after)
         CSR.certificates = [certs[-1]]
```

Comment:
```suggestion
         CSR.certificates = certs[-1:]
```
Does the same and avoids an `IndexError` when there are no certs.